libkbfs/md_journal.go

```diff
@@ -464,6 +464,17 @@ func (j *mdJournal) convertToBranch(
 	j.j = tempJournal
 	j.branchID = bid
 
+	// Garbage-collect the old non-branch entries.  TODO: we'll
```

Comment:
Oh nevermind, I see it.  The "Atomic" one.  Unfortunately it doesn't flush or remove flushed entries.